### ISSUE TITLE
Improve send later

### ISSUE THREAD

**Issue by hfossli** [nylas/N1#2086]
I love Nylas N1 and your new send later feature is great! However I the lack of feedback when using it a little problematic.

Delaying an important reminder for say a year into the future with no receipt is to me problematic. Did server receive it? Can I close the app now or is it still working? I just have to assume stuff and that's not a great feeling. 

Just a reassuring cute animation could be enough and some info on how to view and cancel these postponed mails.

[Screen-recording.mov.zip](https://github.com/nylas/N1/files/240916/Screen-recording.mov.zip)

**Comment by bengotow** [comment]
Hey! You can see postponed emails from the `Drafts` view, but I'll see if we can make this a bit more obvious in the future.

**Comment by hfossli** [comment]
Cool. That wasn't obvious to me. I like that it is available in the `Drafts` views – making it accessible from gmails web interface and other front ends as well.

**Comment by colinking** [comment]
I agree with @hfossli, that it wasn't very obvious to me either. Any chance a "Send Later" tab could be added in the future, that possibly is hidden unless there is at least one email to be sent later?
